multi_physics/QED/python_bindings/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+set(name pxr_qed)
+
+find_package(Python COMPONENTS Interpreter Development REQUIRED)
+find_package(pybind11 CONFIG REQUIRED)
+
+pybind11_add_module(${name} "${name}.cpp")
+
+target_link_libraries(${name} PRIVATE PXRMP_QED)
+
+option(
+    PXRQEDPY_DOUBLE_PRECISION
+    "Compile pxr_qed python module in double precision" ON)
+
+if(PXRQEDPY_DOUBLE_PRECISION)
+    target_compile_definitions(${name} PRIVATE PXRQEDPY_DOUBLE_PRECISION=1)
+endif()
+
+set(PXRQEDPY_UNITS_VALUES SI NORM_OMEGA NORM_LAMBDA HEAVISIDE_LORENTZ)
+set(PXRQEDPY_UNITS SI CACHE STRING "Unit system for the pxr_qed python module (SI/NORM_OMEGA/NORM_LAMBDA/HEAVISIDE_LORENTZ)")
+set_property(CACHE PXRQEDPY_UNITS PROPERTY STRINGS ${PXRQEDPY_UNITS_VALUES})
+
+if(NOT PXRQEDPY_UNITS IN_LIST PXRQEDPY_UNITS_VALUES)
+    message(FATAL_ERROR "PXRQEDPY_UNITS (${PXRQEDPY_UNITS}) must be one of ${PXRQEDPY_UNITS_VALUES}")
+endif()
+
+if(PXRQEDPY_UNITS STREQUAL "SI")
+    target_compile_definitions(${name} PRIVATE PXRQEDPY_SI=1)
+elseif(PXRQEDPY_UNITS STREQUAL "NORM_OMEGA")
+    target_compile_definitions(${name} PRIVATE PXRQEDPY_NORM_OMEGA=1)
+elseif(PXRQEDPY_UNITS STREQUAL "NORM_LAMBDA")
+    target_compile_definitions(${name} PRIVATE PXRQEDPY_NORM_LAMBDA=1)
+elseif(PXRQEDPY_UNITS STREQUAL "HEAVISIDE_LORENTZ")
+    target_compile_definitions(${name} PRIVATE PXRQEDPY_HEAVISIDE_LORENTZ=1)
+else()
+    message(FATAL_ERROR "PXRQEDPY_UNITS is incorrectly defined")
+endif()
+
+set_target_properties(${name} PROPERTIES
+                              INTERPROCEDURAL_OPTIMIZATION ON)
+
+# OpenMP support
+if(PXRMP_QED_OMP)
+   find_package(OpenMP REQUIRED)
+   target_link_libraries(${name} PRIVATE OpenMP::OpenMP_CXX)
+   target_compile_definitions(${name} PRIVATE PXQEDPY_HAS_OPENMP=1)
+endif()
+
+# Move module in "python_bindings" subdirectory
+set_target_properties(${name} PROPERTIES
```

Comment:
Possible, but generally more complex if you want to mirror a good installation directory.
Make this potentially a cached variable, so you can change it from externally.

Example logic for later is in:
- https://github.com/AMReX-Codes/pyamrex or
- https://github.com/openPMD/openPMD-api

Reply:
Would this be OK for now?
```
set(PXRQEDPY_INSTALL_DIR ${CMAKE_BINARY_DIR}/python_bindings CACHE PATH "Installation directory for the python bindings")
set_target_properties(${name} PROPERTIES
    RUNTIME_OUTPUT_DIRECTORY ${PXRQEDPY_INSTALL_DIR})

```

Reply:
Absolutely! :)